js/views/drawerView.js

```diff
@@ -110,7 +111,10 @@ class DrawerView extends Backbone.View {
 
   showDrawer(emptyDrawer, position = null) {
     this.setDrawerPosition(position);
-    this.$el.removeClass('u-display-none').removeAttr('aria-hidden');
+    this.$el
+      .removeClass('u-display-none')
+      .removeAttr('aria-hidden')
```

Comment:
Should this set aria-hidden to false now instead of removing it? To match what's happening with aria-expanded.